src/main/java/com/ibanity/apis/client/products/xs2a/models/create/BatchSynchronizationCreationQuery.java

```diff
@@ -0,0 +1,26 @@
+package com.ibanity.apis.client.products.xs2a.models.create;
+
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+
+import java.time.Instant;
+import java.util.Map;
+
+import static java.util.Collections.emptyMap;
+
+@Getter
+@Builder
+@ToString
+@EqualsAndHashCode
+public class BatchSynchronizationCreationQuery {
+
+    private String[] subtypes;
```

Comment:
We try to use List for collections.

Reply:
@Builder.Default
    private List<String> subtypes = Collections.emptyList();

Reply:
done, except "default", because it's a mandatory field which needs to be set by the dev

Reply:
redone